input/docs/running-builds/runners/cake-frosting.md

```diff
@@ -1,14 +1,19 @@
 Order: 20
 ---
 
+Cake Frosting is a .NET host which allows you to write your build scripts as a console application
+
 :::{.alert .alert-success}
 A console application has the advantage of full IDE support, like IntelliSense, refactoring and debugging.
 :::
 
+:::{.alert .alert-info}
+See [Setting Up A New Frosting Project](/docs/getting-started/setting-up-a-new-frosting-project) for a tutorial how to get started with Cake Frosting.
```

Comment:
```suggestion
See the [Setting Up A New Frosting Project](/docs/getting-started/setting-up-a-new-frosting-project) for a tutorial how to get started with Cake Frosting.
```

Reply:
Sounds wrong to me. This is not about a project, but about a tutorial. Project is only part of the tutorial title.  `See Setting Up A New Frosting PRoject tutorial for how to get started with Cake Frosting` would work.

Reply:
Yes, agreed, re-read this sentence again.  `See Setting Up A New Frosting Project tutorial for how to get started with Cake Frosting` makes sense to me.